var/spack/repos/builtin/packages/py-pennylane-lightning-kokkos/package.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyPennylaneLightningKokkos(CMakePackage, PythonExtension, CudaPackage, ROCmPackage):
+    """The PennyLane-Lightning-Kokkos plugin provides
+    a fast state-vector simulator with Kokkos kernels."""
+
+    homepage = "https://docs.pennylane.ai/projects/lightning-kokkos"
+    git = "https://github.com/PennyLaneAI/pennylane-lightning-kokkos.git"
+    url = "https://github.com/PennyLaneAI/pennylane-lightning-kokkos/archive/refs/tags/v0.29.1.tar.gz"
+
+    maintainers("vincentmr")
```

Comment:
@vincentmr and @mlxd, could you please double-check if we'll proceed with this maintainers list?